### ISSUE TITLE
v1.11.2-221-g5d4d8ab41 ver.today. android (vulkan)stop everygame

### ISSUE THREAD

**Issue by tee4341** [hrydgard/ppsspp#14218]
not load stage open games. not start many game. not open many stage (vulkan) please up date rebug ver. ot update normally ver. on new word  /// it I  can install overwrite on android ///
thankkkk

**Comment by tee4341** [comment]
![Screenshot_20210225173654](https://user-images.githubusercontent.com/48321898/109141599-7e20b200-7790-11eb-9829-94e2a769e454.jpg)

**Comment by Panderner** [comment]
@tee4341 what device are you using?

**Comment by ANR2ME** [comment]
Also which game that have issue?
MGS:PW worked fine on v1.11.2-221-g5d4d8ab41 on my android device tho (Realme C12, Android 10)

**Comment by IrfanH495** [comment]
https://github.com/hrydgard/ppsspp/issues/14211

**Comment by IrfanH495** [comment]
Hatsune miku 2,extend & musou orochi 2 sp ,tenchu san & kurenai, higurashi day fc for vulkan..

**Comment by IrfanH495** [comment]
Hatsune miku 2,extend & musou orochi 2 sp ,tenchu san & kurenai, higurashi day fc for vulkan..

**Comment by tee4341** [comment]
my device is lg g6

**Comment by tee4341** [comment]
please update  workfine ver. (1.11.2=222) on today please i can,t uninstall

**Comment by ANR2ME** [comment]
Do you remember the last working version?

**Comment by IrfanH495** [comment]
Sharp SH-04H
Os 8.0.0
Snapdragon 820
Adreno 530
this issue comes in version v1.11.2-191-g7cefafeb3
for the previous version I don't know.

**Comment by tee4341** [comment]
I can,t remember l work fine  ver. help me please!!!

**Comment by Panderner** [comment]
> I can,t remember l work fine ver. help me please!!!

@tee4341 have you tried to use stable version of PPSSPP?

**Comment by IrfanH495** [comment]
> I can,t remember l work fine ver. help me please!!!

v1.11.2-87-ga911d99ac working for me

**Comment by IrfanH495** [comment]
Fc
https://drive.google.com/file/d/11Uzq3ZCXTdjqEkgNTsDvbejuneoJnR8U/view?usp=drivesdk
Setting
https://drive.google.com/file/d/11kfUSacMErkgQWqV9tKrkDc9jpXTqkrY/view?usp=drivesdk

**Comment by tee4341** [comment]
thank IrfanH495

**Comment by ANR2ME** [comment]
> 
> 
> thank IrfanH495 but I don't need uninstall ppsspp need ver new can overwrite

The devs need to know which version was the last working one, so it would be nice if you can uninstall the broken one and try an older one that still worked, so the devs can fix it sooner :)

PS: May be because of volatile memory lock issue, similar to the crash on GTA games

**Comment by IrfanH495** [comment]
> thank IrfanH495 but I don't need uninstall ppsspp need ver new can overwrite
What.  you can't uninstall it.  try deleting ppsspp data & cache after rebooting your device then uninstalling it, make sure to force stop ppsspp from the app info before rebooting.  maybe it can help, only once I got the message this app is dangerous, now I don't get any more messages.

**Comment by IrfanH495** [comment]
> > thank IrfanH495 but I don't need uninstall ppsspp need ver new can overwrite
> 
> The devs need to know which version was the last working one, so it would be nice if you can uninstall the broken one and try an older one that still worked, so the devs can fix it sooner :)
> 
> PS: May be because of volatile memory lock issue, similar to the crash on GTA games

unknownbrackets he told me to use ver.1.11.2-121-gc190abc57, and it worked for me, I don't know the version above is working or not.
I can't help much, my quota is dying.

**Comment by IrfanH495** [comment]
> > thank IrfanH495 but I don't need uninstall ppsspp need ver new can overwrite
> 
> The devs need to know which version was the last working one, so it would be nice if you can uninstall the broken one and try an older one that still worked, so the devs can fix it sooner :)
> 
> PS: May be because of volatile memory lock issue, similar to the crash on GTA games

ppsspp-v1.11.2-184-ga062c177a-android.apk work for me

ppsspp-v1.11.2-191-g7cefafeb3-android.apk not work 
maybe this issue is above 184. i will try again later

**Comment by IrfanH495** [comment]
ppsspp-v1.11.2-189-gad3d23424-android no work for me vukan still fc

**Comment by IrfanH495** [comment]
@ANR2ME @Panderner 
ppsspp-v1.11.2-186-g90f1e98ab- masalahnya disini sampai bulid terbaru
the problem is here until the latest build

**Comment by IrfanH495** [comment]
ppsspp-v1.11.2-184-ga062c177a-android.apk work for me
![Screenshot_20210225-222421](https://user-images.githubusercontent.com/78854859/109175439-5fceac80-77b8-11eb-942d-2848a3a009d8.png)
![Screenshot_20210225-222434](https://user-images.githubusercontent.com/78854859/109175469-652bf700-77b8-11eb-8639-077bd3a56508.png)
![Screenshot_20210225-222344](https://user-images.githubusercontent.com/78854859/109175470-65c48d80-77b8-11eb-8c08-0049313594ce.png)

**Comment by ANR2ME** [comment]
> 
> 
> ppsspp-v1.11.2-189-gad3d23424-android no work for me vukan still fc

only crashed on vulkan? or opengl fc too?

If it's started from v1.11.2-186-g90f1e98ab it might be related to https://github.com/hrydgard/ppsspp/pull/14195

**Comment by IrfanH495** [comment]
> > ppsspp-v1.11.2-189-gad3d23424-android no work for me vukan still fc
> 
> only crashed on vulkan? or opengl fc too?
> 
> If this issue started from v1.11.2-186-g90f1e98ab it might be related to #14195

Opengl fine
Vulkan FC

**Comment by IrfanH495** [comment]
Vulkan only

**Comment by unknownbrackets** [comment]
I can't seem to reproduce any problems on my device.

Based on the range, I suppose it must be glslang.  Maybe it's using newer spirv than the device supports?

Is this while using any post-processing shaders?

-[Unknown]

**Comment by hrydgard** [comment]
Most likely some driver bug with some SPIR-V that the new glslang likes to produce but old one didn't.... Nasty.

We could of course simply downgrade glslang again. Though would be a shame to fall behind too much, there's always build fixes etc even though we don't really need any new features...

I'm gonna see if I can repro on any of my devices soon. I don't have a lot of 5xx Adrenos though...

**Comment by unknownbrackets** [comment]
Just to note: my Pixel has a 821/Adreno 530 (exact same as LG G6) and isn't affected, so presumably this must be related to Android version.  Definitely tricky if it's only Adreno 5xx devices on older Android...

Most likely it'd help to see an adb logcat fro an affected device when it crashes:
https://github.com/hrydgard/ppsspp/wiki/How-to-get-an-adb-logcat-from-a-PC

-[Unknown]

**Comment by Saramagrean** [comment]
Here, Logcat :)
[ppsspplog.txt](https://github.com/hrydgard/ppsspp/files/6060814/ppsspplog.txt)

Devices: Redmi Note 8 (Snapdragon 665, Adreno 610, Ram 3, Android 10) and Xiaomi Black Shark (Snapdragon 845, Adreno 630)

**Comment by IrfanH495** [comment]
I can't help my device is damaged in the USB section
![Screenshot_20210301-183813](https://user-images.githubusercontent.com/78854859/109492304-58abe500-7abd-11eb-854e-269ccf5d53f3.png)

**Comment by IrfanH495** [comment]
not all games are closed, for example in ppsspp-v1.11.2-184-ga062c177a- everything is still good.  but if I use ver ppsspp-v1.11.2-186-g90f1e98ab- some games stop and some graphic games get bad.  so close issue for naruto and the wariors. https://github.com/hrydgard/ppsspp/issues/14211

**Comment by Saramagrean** [comment]
After update to Android 11 (Vulkan driver v1.1.128), I can't get crash anymore.

**Comment by ANR2ME** [comment]
Nowadays, GPU driver can be updated through official app from their manufacturer, some available at Playstore, some at other App market bundle with their OS.

So may be try installing those GPU driver update to see whether it solves this kinda issue?

**Comment by IrfanH495** [comment]
I will try it

**Comment by IrfanH495** [comment]
I did not find it, what is the name of the application

**Comment by ANR2ME** [comment]
> I did not find it, what is the name of the application

usually i just search for "gpu driver"

**Comment by IrfanH495** [comment]
> > I did not find it, what is the name of the application
> 
> usually i just search for "gpu driver" at Playstore and find the one that match my gpu or it's manufacturer.
> 
> But for LG G6 probably need TWRP https://forum.xda-developers.com/t/new-adreno-530-gpu-driver-for-our-g6.3832403/

what it must be root.

**Comment by IrfanH495** [comment]
it's okay, I will use ppsspp v1.11.2-184-ga062c177a- so far you developers have worked hard, to solve the issues that arise, I myself do not mind this issue.  
it's okay you don't need to hurry, on this issue.  because I know being a developer is not easy. one issue is over, a new issue will come.  
I hope that in version 1.12.0 everything returns to normal.

**Comment by hrydgard** [comment]
Yeah, this does seem like a serious issue. Will have to solve it one way or another before 1.12.

**Comment by Saramagrean** [comment]
Since v1.11.2-312-g38e384d83, I can use Vulkan without any crash maybe it fixed by https://github.com/hrydgard/ppsspp/pull/14273?

**Comment by hrydgard** [comment]
Oh, maybe so! Let's see if @tee4341 reports back.

**Comment by IrfanH495** [comment]
@hrydgard thanks ppsspp v1.11.2-382-gee641c1ed
Work for me bolt vulkan & open gl, but background sometimes turns blue and that's not a problem.
![Screenshot_20210401-183728](https://user-images.githubusercontent.com/78854859/113289715-c7799980-931a-11eb-81a1-fadaa3c20670.png)
![Screenshot_20210401-183941](https://user-images.githubusercontent.com/78854859/113289725-ce081100-931a-11eb-8e71-69de65b3068d.png)
You can close this issue, thank you

**Comment by unknownbrackets** [comment]
Glad it's fixed.  Not sure what would cause the background to randomly stop animating... there's a setting to enable/disable the animation now.

-[Unknown]

**Comment by ANR2ME** [comment]
Btw is there a way to show the country flag on recent history on Windows like that screenshot? 
it's pretty confusing if we have multiple versions of the game with different region each, while the icon usually the same.

**Comment by unknownbrackets** [comment]
Go to another tab and click the settings gear icon (not Settings), then you can choose that.

-[Unknown]
